osutil/kcmdline/kcmdline.go

```diff
@@ -473,3 +473,18 @@ func (kap *ArgumentPattern) UnmarshalYAML(unmarshal func(interface{}) error) err
 	}
 	return kap.unmarshalFromString(arg)
 }
+
+// FilterOut parses a kernel command line and remove arguments that
+// matches match patterns.
+func FilterOut(cmdline string, match []ArgumentPattern) []string {
```

Comment:
`FilterOut` feels a bit incomplete for a function name. How about `ExcludeMatchingFilter` or `RemoveMatchingFilter` instead?